### PR TITLE
Create Left Sidebar layout

### PR DESCRIPTION
**Pull request by negreirosleo**
This PR implements the Left Sidebar for the Layout.
- When it is expanded it shows the full logo with text, the link texts and the horizontal dark-mode switch
- When it is collapsed it shows the small logo, the navigation icons only and the vertical dark-mode switch

[Screencast from 2023-09-18 16-29-29.webm](https://github.com/Igalia/phpreport/assets/20492786/de653486-dcc1-4ded-acda-93565e0a594b)

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -9,6 +9,10 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
```
Comment (anarute) [inline frontend/package.json:13]: Out of curiosity, what are these dependencies for?

**Review by anarute — APPROVED** [review]
Overall it looks good to me!

File: frontend/package.json (hunk 1 above)
Comment (negreirosleo) [reply frontend/package.json:13]: These are part of a CSS-in-Js lib that Joy-UI requires to work

File: frontend/src/app/auth/AuthorizedPage.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    return <div>Loading...</div>
+  }
+
+  if (error) {
+    return <div>Oops... {error.message}</div>
+  }
+
+  if (!isAuthenticated) {
+    return <button onClick={() => signinRedirect()}>BOTAO</button>
+  }
```
Comment (anarute) [inline frontend/src/app/auth/AuthorizedPage.tsx:32]: Not sure if this was left on purpose or not but thought it worth to ask :)

**Review by dmtrek14 — COMMENTED** [review]
Overall, this is looking great. I have a number of small accessibility related changes marked throughout. 

One small nit: the focus border on the menu items seems to get cut off on the sides (in both Chromium and Firefox). Can you see if you can fix it? 

Screenshot from 2023-09-19 10-02-38

**Review by dmtrek14 — APPROVED** [review]
Thanks for the latest round of changes - this all looks good to me.

**Pull request merged**